src/ModelBody.cpp

```diff
@@ -133,10 +133,17 @@ void ModelBody::SetStatic(bool isStatic)
 void ModelBody::SetColliding(bool colliding)
 {
 	m_colliding = colliding;
-	if (colliding)
+	if (colliding) {
 		m_geom->Enable();
-	else
+		for(auto &g : m_dynGeoms) {
+			g->Enable();
+		}
```

Comment:
I'm curious if this solves the recent batch of issue reports regarding inability to collide with / land on stations. I did make the CollisionSpace code stricter, as it no longer includes disabled Geoms in the BVH construction at all (rather than filtering them out later in the pipeline).

Reply:
Added these changes to the master and tested #5705 - does not help.